tests/http-adapter.spec.js

```diff
@@ -33,166 +29,252 @@ describe('HTTP adapter', function () {
     })
   })
 
-  describe('#get', function () {
-    it('returns a parsed resource from the network', function () {
+  describe('#get', () => {
+    it('returns a parsed resource from the network', async done => {
```

Comment:
I think just making these `async` and `await`-ing inside obviates the need for `done`...? Did you find otherwise?